packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -139,6 +139,26 @@ class FlutterDevice {
     return reports;
   }
 
+  // Lists program elements changed in the most recent reload that have not
+  // since executed.
+  Future<List<ProgramElement>> unusedChangesInLastReload() async {
+    final List<Future<List<ProgramElement>>> reports =
+        <Future<List<ProgramElement>>>[];
```

Comment:
Per https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#prefer-line-length-of-80-characters the guideline is to indent by 2.